frontends/web/src/components/icon/logo.tsx

```diff
@@ -56,23 +56,22 @@ import PAXG_GREY from './assets/paxg-white.svg';
 import ShiftLogo from './assets/shift-cryptosecurity-logo.svg';
 import style from './logo.module.css';
 
-interface GenericProps {
-    [property: string]: any;
-}
```

Comment:
removed and replaced by `type ImgProps = JSX.IntrinsicElements['img'];` as we already do in icons.tsx